manage.py

```diff
@@ -58,6 +59,7 @@ def deploy():
     analyze_all()
 
 
+@boto3_mocks.restore_http_adapter
```

Comment:
I assume you have, but did you try importing all of the the moto mocks as decorators, like:

`from moto import mock_cloudwatch, mock_s3, mock_lambda, mock_sqs`

and wrapping the whole `MainTest` class with them, like:

```
@mock_s3
@mock_sqs
@mock_lambda
@mock_cloudwatch
class MainTest(unittest.TestCase):
    ....
```

instead of instantiating them within the test case setup?